tinywallet/tinywallet/screens.py

```diff
@@ -1869,6 +1883,50 @@ def __init__(self, saveName):
         bttn.clicked.connect(self.nameChangeClicked)
         grid.addWidget(bttn, row, 1)
 
+        # RELAY FEE
+
+        # Set string constants.
+        self.lowStr = "low"
+        self.defaultStr = "default"
+        self.highStr = "high"
+
+        # Set values that are considered low, default, and high.
+        self.feeRates = {
+            self.lowStr: 4000,
+            self.defaultStr: int(DefaultRelayFeePerKb),
+            self.highStr: 114000,
+        }
+        self.feeLvl = ""
+
+        # Find the current level. Warn if the current level isn't one of the
+        # three.
+        for k, v in self.feeRates.items():
+            if v == relayFee:
+                self.feeLvl = k
+                break
+        else:
+            log.warn(f"non standard relay fee set: {relayFee}")
+
+        # Helper to check the current value.
+        def setChecked(btn):
+            if self.feeLvl == btn.text():
+                btn.setChecked(True)
+
+        row += 1
+        grid.addWidget(Q.makeLabel("Relay Fee", 14, Q.ALIGN_LEFT), row, 0)
+        row += 1
+        btn1 = QtWidgets.QRadioButton(self.lowStr)
```

Comment:
I'm not sure about the default font on the radio button. I think you can pass the button directly to `Q.setProperties` to play with the font properties. Make sure it's using `fontFamily="Roboto"`. Something seems off about it.